sklearn/preprocessing/tests/test_polynomial.py

```diff
@@ -465,10 +465,6 @@ def test_spline_transformer_n_features_out(n_knots, include_bias, degree):
 @pytest.mark.parametrize(
     "params, err_msg",
     [
-        ({"degree": -1}, "degree must be a non-negative integer"),
-        ({"degree": 2.5}, "degree must be a non-negative int or tuple"),
-        ({"degree": "12"}, r"degree=\(min_degree, max_degree\) must"),
-        ({"degree": "string"}, "degree must be a non-negative int or tuple"),
```

Comment:
can you add the case where one put a tuple with 3 values to check that fall into the right exception branch.